workspace.json

```diff
@@ -7990,6 +7990,12 @@
             "buildTarget": "services-notifications:build"
           }
         },
+        "worker": {
+          "builder": "@nrwl/workspace:run-commands",
+          "options": {
+            "command": "yarn ts-node -P apps/services/notifications/tsconfig.app.json apps/services/notifications/src/worker.ts"
+          }
+        },
```

Comment:
I don't think this will work well with deployments. Not sure if you've checked with Andes, but I think this will either need to be a standalone "app" (using shared models/code like the auth services), or extra process arguments to the "main.ts" file.

We could maybe expand infra-nest-server#bootstrap to support workers or arbitrary commands when the process is started with extra arguments, eg `node main.js --worker`.

Reply:
yes, that's right. We need to iterate on this part a bit

Reply:
@eirikurn this is only for local development while we are iterating on this. we have a separate pull request for the infra setup, which is very much a draft, and waiting for this one, where we will take the next steps forward with andes

Reply:
Sure. As discussed on slack, I recommend starting the worker from `main.ts`, and use a `@nrwl/node:execute` builder with `"args": "--something"`.

Reply:
yup, it's been updated and docs have been added to indicate.